packages/core/src/runtime/App.tsx

```diff
@@ -10,10 +10,8 @@ import { HelmetProvider } from 'react-helmet-async';
 import React, { useContext, useLayoutEffect } from 'react';
 import { Header, PageData, cleanUrl, MDX_REGEXP } from '@rspress/shared';
 import globalComponents from 'virtual-global-components';
-import 'virtual-global-styles';
-
-// eslint-disable-next-line import/no-commonjs
-const { default: Theme } = require('@theme');
+import Theme from '@theme';
+import('virtual-global-styles');
```

Comment:
Is dynamic import used to re-order the CSS?

Reply:
The position of`import('virtual-global-styles')` should be after @theme import to avoid the latter overrides the custom global styles.

Reply:
ok, can we just use a sync import: `import 'virtual-global-styles';`